src/workflow/Util.cpp

```diff
@@ -171,27 +172,10 @@ bool mergeOutputTableSummaryReports(IdfObject& existingObject, const IdfObject&
 }
 
 bool addEnergyPlusOutputRequest(Workspace& workspace, IdfObject& idfObject) {
```

Comment:
I was super tempted to add a `LOG` (Warn or Info) in there, something along the lines of

if iddObjectType.valueName does not start with "Output" or "Meter", or "EnergyManagement" or "PythonPlugin", then issue a warning like "Requested to add an object of type XXX, ensure you aren't going to modify the energy usage of the model".

But I'm guessing @eringold and @shorowit wouldn't like that too much either, right?

Reply:
I'm not against log messages. Though technically objects _starting with_ 'EnergyManagement' or 'PythonPlugin' could modify energy use. Maybe better would be to check if object type names _contain_ just 'Output' or 'Meter'.

Reply:
This is true PythonPlugin (and EnergyManagement is the same) were some of the objects that you specifically wanted in your OP, and they **could** have a valid application.

Reply:
I decided to add the Warn after all, but not being restrictive and letting the request be processed anyways.

cf https://github.com/NREL/OpenStudio/pull/5358#pullrequestreview-2698044148